app/views/govuk_publishing_components/components/_intervention.html.erb

```diff
@@ -16,14 +16,31 @@
 
   intervention_helper = GovukPublishingComponents::Presenters::InterventionHelper.new(local_assigns)
   dismiss_href = intervention_helper.dismiss_link
+
+  suggestion_tag_options = {
+    class: "govuk-link gem-c-intervention__suggestion-link",
```

Comment:
Is the `gem-c-intervention__suggestion-link` class necessary? It doesn't seem to add any styles.

Reply:
Interesting. I don't know if we do or not really as it's what Max added in and I've only moved it to this hash. I'm open to removing it and seeing what happens as a case for making things tidy, but it might be a little out of scope for this PR otherwise.

Reply:
The only place the class appears is in tests, so removing it would break those. But it isn't adding any styling. Agree, this is outside the scope of this PR, but can you raise an issue about it? Thanks.

Reply:
Done